tests/test_response.py

```diff
@@ -107,6 +107,34 @@ def test_response_spec():
     assert spec.get(404) is None
 
 
+def test_response_spec_with_schema_extra():
```

Comment:
if possible, can you use the fixture `snapshot_json` and compare the whole payload instead of cherrypicked parts? See example here:

https://github.com/0b01001001/spectree/blob/0508b707811a85e8294afa4a14fedd6b9b66ba57/tests/test_plugin.py#L25-L34


to generate the snapshot, add the following parameter: `pytest --snapshot-update`